cassiopeia/tools/branch_length_estimator/IIDExponentialMLE.py

```diff
@@ -52,6 +56,7 @@ class IIDExponentialMLE(BranchLengthEstimator):
     def __init__(
         self,
         minimum_branch_length: float = 0.01,
+        site_rates: Optional[List[float]] = None,
```

Comment:
We need to change the docstring of the `mutation_rate` attribute to clarify that it is a list if site rate variation is used, and a float otherwise.

Reply:
Finally, a better name might be `relative_mutation_rates` (firstly, because `relative` is important here, and secondly, to keep the naming compatible with the previously existing `mutation_rate` attribute).